src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_params.py

```diff
@@ -104,9 +108,14 @@ def load_arguments(self, _):
 
     with self.argument_context('backup protection disable') as c:
         c.argument('delete_backup_data', arg_type=get_three_state_flag(), help='Option to delete existing backed up data in the Recovery services vault.')
+
+    with self.argument_context('backup protection isenabled-for-vm') as c:
+        c.argument('vm_id', help='ID of the virtual machine to be checked for protection.')
+
     # Restore
+    # TODO: Need to use recovery_point.id
```

Comment:
? Lots of TODOs here.

Reply:
Please refer to the comment above. In this case, in future, we want to be able to fetch the item_name and container_name from recovery_point.id using --ids.

Reply:
Please post the issue regarding the system not recognizing your resource ID here:
https://github.com/Azure/msrestazure-for-python

Then just reference that issue in your TODOs so it's clear why these are not being resolved now (and to make it clearer what needs to be changed when the issue is fixed).